api/v1alpha1/workspace_validation.go

```diff
@@ -86,7 +83,18 @@ func (w *Workspace) validateUpdate(old *Workspace) (errs *apis.FieldError) {
 	return errs
 }
 
-func (r *TuningSpec) validateCreate() (errs *apis.FieldError) {
+func (r *TuningSpec) validateCreate(ctx context.Context) (errs *apis.FieldError) {
+	methodLowerCase := strings.ToLower(string(r.Method))
+	if methodLowerCase != string(TuningMethodLora) && methodLowerCase != string(TuningMethodQLora) {
+		errs = errs.Also(apis.ErrInvalidValue(r.Method, "Method"))
+	}
+	if r.Config == "" {
+		errs = errs.Also(apis.ErrMissingField("Config"))
+	} else {
+		if err := r.validateConfigMap(ctx, methodLowerCase); err != nil {
+			errs = errs.Also(apis.ErrGeneric(fmt.Sprintf("Failed to evaluate validateConfigMap: %v", err), "Config"))
```

Comment:
If user specifies the configmap template, the template should be inside the workspace CR's namespace, not the release namespace.

Reply:
Yes good point, if user uses a template - configmap will exist in both release and workspace namespace (configmap can only be mounted to a pod in the same namespace). 

If a user specifies own template - they are responsible for creating configmap in advance in workspace namespace.